addons/web/static/src/views/kanban/kanban_controller.scss

```diff
@@ -384,6 +385,8 @@
 
         &:focus-visible,
         &:focus-within {
+            z-index: 2;
```

Comment:
I'm wondering why specifically "2" ? Hover what should it be elevated ?

Reply:
Hello, it's to elevate the outline on focus state above the next kanbancard, otherwise the bottom side of the outline is hidden behind the next `o_kanban_record`, the "2" comes to be elevated above the `o_kanban_header` which has a z-index-1 (else it would display the folded column title above the kanban_card on drag) 
![image](https://github.com/odoo/odoo/assets/118886338/c4d6f8bb-6023-4892-bc1c-5eb57c3cabf0)
<img width="1156" alt="image" src="https://github.com/odoo/odoo/assets/118886338/d22999ba-b2d9-48ce-a608-2fa2f8313ffa">

Reply:
Yes, that's what I supposed to.

I have just seen @stefanorigano 's comment [1] about having a z-index mapping (so we could have something like `calc(kanban-header-zindex + 1)`) and I agree. But as this is in stable, maybe keep this idea for master...

[1]: https://github.com/odoo/odoo/pull/143345#discussion_r1409492847